### PR TITLE
feat: redesign capturing claim type (CMC-358)

### PR DESCRIPTION
**Pull request by pd-dudkiewicz**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CMC-358

### Change description ###

- changed claim type selection from dropdown list to radio list
- added optional page for selecting personal injury type

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/unspec/callback/CallbackType.java [hunk 1]
```diff
@@ -12,7 +12,8 @@
     ABOUT_TO_SUBMIT("about-to-submit"),
     SUBMITTED("submitted"),
     MID("mid"),
-    MID_SECONDARY("mid-secondary");
+    MID_SECONDARY("mid-secondary"),
+    MID_TERTIARY("mid-tertiary");
```
Comment (sabahirfan) [inline src/main/java/uk/gov/hmcts/reform/unspec/callback/CallbackType.java]: seems time to enhance our framework.

**Review by dharmendrak — DISMISSED** [review]
except additonal enum ` MID_TERTIARY("mid-tertiary");`

**Review by dharmendrak — APPROVED** [review]
LGTM

File: ccd-definition/CaseField.json [hunk 2]
```diff
@@ -63,11 +63,36 @@
...
+    "FieldTypeParameter": "PersonalInjuryType",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "ID": "personalInjuryTypeOther",
+    "Label": "Enter personal injury type",
+    "FieldType": "Text",
+    "SecurityClassification": "Public"
```
Comment (AlisonRevittHM) [inline ccd-definition/CaseField.json:94]: are we waiting on the spike to see how this can be modelled generically?

Comment (pd-dudkiewicz) [reply ccd-definition/CaseField.json:94]: I think it was agreed to go with what we have now and maybe change it depending on spike outcome

**Review by douglasrice — APPROVED** [review]

**Pull request merged**